src/wallet/wallet.cpp

```diff
@@ -41,7 +41,7 @@ CAmount maxTxFee = DEFAULT_TRANSACTION_MAXFEE;
 unsigned int nTxConfirmTarget = DEFAULT_TX_CONFIRM_TARGET;
 bool bSpendZeroConfChange = true;
 bool fSendFreeTransactions = false;
-bool fPayAtLeastCustomFee = true;
+bool fPayAtLeastCustomFee = false;
```

Comment:
How this is usually done is to add a hidden command line option to override this setting. This is how most of the tests work that need special setup, for special tests.

On the other hand it is not good to have the entire wallet test depend on a non-standard setting, so then it'd have to be a new test.

Isn't there already a PR to change the default?